libs/java.json

```diff
@@ -247,5 +247,9 @@
   {"id": "java.storio", "imports": ["com.pushtorefresh.storio3."], "name": "StorIO", "repo": "pushtorefresh/storio", "tags": ["android-sql","reactive"], "tech": ["storage"], "status": "awaiting-model"},
   {"id": "java.sqlbrite", "imports": ["com.squareup.sqlbrite3."], "name": "SQL Brite", "repo": "square/sqlbrite", "tags": ["android-sql"], "tech": ["storage"], "status": "awaiting-model"},
   {"id": "java.toothpick", "imports": ["toothpick."], "name": "Toothpick", "repo": "stephanenicolas/toothpick", "tags": ["injection"], "tech": ["app-arch"], "status": "awaiting-model"},
-  {"id": "java.parceler", "imports": ["org.parceler."], "name": "Parceler", "repo": "johncarl81/parceler", "tags": ["android-serialization"], "tech": ["efficient-data"], "status": "awaiting-model"}
+  {"id": "java.parceler", "imports": ["org.parceler."], "name": "Parceler", "repo": "johncarl81/parceler", "tags": ["android-serialization"], "tech": ["efficient-data"], "status": "awaiting-model"},
+  {"id": "java.jpa", "imports": ["javax.persistence."], "name": "JPA", "repo": "eclipse-ee4j/jpa-api", "tags": ["orm"], "tech": ["storage"], "status": "model-ready"},
```

Comment:
status should be 'awaiting-model' for all new entries. this means that we did not train an ML classifier for this library.

Reply:
Done!